### ISSUE TITLE
Use SHAP to rank features from a Keras Sequential model

### ISSUE THREAD

**Issue by somnathrakshit** [shap/shap#286]
How can we use SHAP to rank the features in a Keras Sequential model of an autoencoder? The dataset that I have is tabular in nature containing class labels and is used for multiclass classification.

**Comment by slundberg** [comment]
Usually an auto-encoder tries to reconstruct the input after a dimensionality reduction. How exactly is it being used for classification?

**Comment by somnathrakshit** [comment]
I was using [this](http://ufldl.stanford.edu/wiki/index.php/Stacked_Autoencoders) approach.

**Comment by slundberg** [comment]
Ah. How you pre-train doesn't effect how the final model is interpreted. You can just try to take your final model (that outputs probabilities or logits) and pass it to GradientExplainer like in the example notebooks and it should work. If not feel free to post the code.

**Comment by somnathrakshit** [comment]
Ok, sure. I will try this out.
